rebrand/config/_typography.scss

```diff
@@ -71,6 +71,6 @@ sup {
 
 @mixin fsize-36 {
     font-size: 36px !important;
-    letter-spacing: 0.02em;
+    letter-spacing: 0.08em;
```

Comment:
Compared to the designs, I think we want to stick with 0.02 or maaaaybe 0.04? I think there's differences between how Sketch and browsers render fonts, so the letter spacing and line-height can be a bit tricky to specify!

Reply:
I've fixed that - do you want me to install onto mjolnir-templates/web or do you think it's not necessary?

Reply:
What's your argument for not doing? I would do for completeness, but I could be swayed!

Reply:
Now I've reverted the letter spacing change, only the colours have changed. We know the letter-spacing of that mixin is fine cos we use it on live, so really you're only just checking the colours now on the previews :)